iscan/models.py

```diff
@@ -646,10 +667,22 @@ def import_corpus(self):
             else:
                 return False
             c.load(parser, self.source_directory)
+        spade_conf = self.configuration_data
+
+        if "sibilant_segments" in spade_conf:
```

Comment:
These lines are probably going to throw an error (since you're assigning values to a property.  Just delete all of these and it should be fine.